package-lock.json

```diff
@@ -3487,6 +3487,7 @@
       "version": "3.2.1",
       "resolved": "https://registry.npmjs.org/ansi-styles/-/ansi-styles-3.2.1.tgz",
       "integrity": "sha512-VT0ZI6kZRdTh8YyJw3SMbYm/u+NqfsAxEpWO0Pf9sq8/e94WxxOpPKx9FR1FlyCtOVDNOQ+8ntlqFxiRc+r5qA==",
+      "dev": true,
```

Comment:
All of these `"dev": true` are telling - sanitize-html had dependencies that were generally only dev dependencies... but we were adding them into our app.  Ouch.